store/datastore/users.go

```diff
@@ -28,39 +29,49 @@ func (db *datastore) GetUserList() ([]*model.User, error) {
 func (db *datastore) GetUserFeed(listof []*model.RepoLite) ([]*model.Feed, error) {
 	var (
 		args []interface{}
-		stmt string
 		err  error
-
-		feed = []*model.Feed{}
+		feed = model.Feeds{}
 	)
-	switch meddler.Default {
-	case meddler.PostgreSQL:
-		stmt, args = toListPosgres(listof)
-	default:
-		stmt, args = toList(listof)
+	_stmt, _args := toList(listof)
+
+	for i, stmt := range _stmt{
+		args = _args[i]
+		if len(args) > 0 {
+			var _feed []*model.Feed
+			err = meddler.QueryAll(db, &_feed, fmt.Sprintf(userFeedQuery, stmt), args...)
+			if err != nil{
+				break
+			}
+			feed = append(feed, _feed...)
+		}
 	}
-	if len(args) > 0 {
-		err = meddler.QueryAll(db, &feed, fmt.Sprintf(userFeedQuery, stmt), args...)
+	sort.Sort(sort.Reverse(feed))
```

Comment:
we can probably return `feed` if `len(feed) <= 50`. The majority of drone users don't have > 999 repos so we can optimize for this case